grunttasks/eslint.js

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  grunt.config('eslint', {
+    options: {
+      eslintrc: '.eslintrc'
+    },
+    files: [
+      '{,bin/,config/,grunttasks/,lib/**/,test/,test/lib/mocks}*.js'
```

Comment:
test/lib/blanket.js is not our code so I excluded it from the checks, particularly to prevent being nagged about a missing MPL license header.  I wonder if this fantastical glob syntax has a way to say "not this file" which would make it clearer...